RELEASES.md

```diff
@@ -77,18 +77,30 @@ Support horizons will be defined corresponding to a release branch, identified
 by `<major>.<minor>`. Releases branches will be in one of several states:
 
 - __*Next*__: The next planned release branch.
-- __*Active*__: The release branch is currently supported and accepting patches.
+- __*Active*__: The release is a stable branch which is currently supported and accepting patches.
```

Comment:
This phrasing seems a little awkward to me. Maybe something like `The most recent release is a stable branch...`?

Reply:
On reading more of the context around I think this is okay, so nevermind.